worlds/witness/Options.py

```diff
@@ -158,6 +158,12 @@ class HintAmount(Range):
     default = 10
 
 
+class DeathLink(Toggle):
+    """If on: Whenever you fail a puzzle (with some exceptions), everyone dies.

```

Comment:
not everyone, just everyone also on death link

Reply:
Sure. I will find which game's description I referenced for that, so you can change that one too :D

Reply:
![image](https://user-images.githubusercontent.com/57900059/223865506-494e79fa-358a-43c1-8f00-8600bc84e441.png)

Reply:
Super Mario 64